content/applications/hr/lunch/management.rst

```diff
@@ -0,0 +1,158 @@
+==============
+Managing Lunch
+==============
+
+In Odoo's *Lunch* application, it is required to have someone manage the orders, vendors, and
+products. In addition, someone must be responsible for the orders, and notifying employees when
+their orders have arrived. This can be the same person.
+
+In order to manage the *Lunch* application, the user must have :guilabel:`Administrator` rights set
+for the :guilabel:`Lunch` application. This is verified by navigating to :menuselection:`Settings
+app --> → Manage Users`. Then click on a user to view the various settings. For more information on
+access rights, refer to the :doc:`Access rights </applications/general/users/access_rights/>`
+documentation.
+
+.. note::
+   Only users with administration rights are able to view anything other than :guilabel:`My Lunch`
+   in the *Lunch* app.
+
+.. _lunch/todays-orders:
+
+Today's Orders
+==============
+
+To view and manage the orders for the day, navigate to :menuselection:`Lunch app --> Manager:
+Today's Orders`. All orders for the day are presented in a list view on the :guilabel:`Today's
+Orders` dashboard, with a filter for :guilabel:`Today`, and grouped by :guilabel:`Vendor`, by
+default.
+
+The :guilabel:`Order Date`, :guilabel:`Vendor`, :guilabel:`Product`, :guilabel:`Extras`,
+:guilabel:`Notes`, :guilabel:`User`, :guilabel:`Lunch Location`, :guilabel:`Price`,
+:guilabel:`Status`, and :guilabel:`Company` information appear in the list.
+
+.. image:: management/today.png
+   :align: center
+   :alt: The list that appears in the Today's Orders dashboard, with the filters and top column
+         names highlighted.
+
+.. _lunch/cancel:
+
+Cancel Orders
+-------------
+
+If an order needs to be canceled and should not be sent to the vendor, individual products can be
+canceled one at a time.
+
+From the :guilabel:`Today's Orders` dashboard, a :guilabel:`✖️ Cancel` icon appears at the end of
+each product line that has a red :guilabel:`Status` tag of :guilabel:`Ordered`. If an item needs to
+be canceled, click the :guilabel:`✖️ Cancel` icon to cancel the order for that single product.
+
+.. image:: management/cancel.png
+   :align: center
+   :alt: Order lines with the cancel icon highlighted.
+
+.. _lunch/send-orders:
+
+Send Orders
+-----------
+
+The first step in managing the *Lunch* app is to send the orders to the vendors.
```

Comment:
Is this always the first step? If so, I'd put this section above the "Cancel Orders" step

Reply:
I left the cancel above it because you need to cancel items before they are sent to the vendor. Also, anyone can cancel their own order- not just managers. I added a line to that affect in the cancel section, so this is the first thing for managers to acutally manage and send/receive orders.